### DISCUSSION TITLE
what is dual mode for in addition to dark/light modes

### DISCUSSION THREAD

**Discussion by copdips** [pradyunsg/furo#543] [Questions] [answered]
### What's happening?

Hello,

In the dark/light mode switch, it has a third choice like: 
![image](https://user-images.githubusercontent.com/13083856/193569710-eb93c85c-9a11-4357-9b11-24743407a832.png)

After searching in the [issues](https://github.com/pradyunsg/furo/issues/24), it seems that this is called `dual` mode.

Anyone could give a brief description, what is it for? 
Because it seems that it has the same effet when I choose dark mode or dual mode.

### Reproducer

switch between dark mode and dual mode

### Expectation

1. more explanations in the doc
2. options to disable the dual mode, sometimes, dark/light is enough.

### Code of Conduct

- [X] I agree to follow the [Code of Conduct](https://github.com/pradyunsg/furo/blob/main/CODE_OF_CONDUCT.md).

**Comment by pradyunsg** [comment, marked as answer]
https://pradyunsg.me/furo/customisation/colors/#how-light-and-dark-mode-work

It respects the dark/light mode as requested by the user’s browser.

**Reply by pradyunsg** [reply]
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme

**Comment by copdips** [comment]
@pradyunsg 
ah okey, it means auto mode based on user preference, thanks a lot for your apply
